app/main/views/questionnaire.py

```diff
@@ -96,6 +96,19 @@ def get_summary(eq_id, form_type, collection_id):
     return redirect_to_questionnaire_page(eq_id, form_type, collection_id, latest_location)
 
 
+@questionnaire_blueprint.route('confirmation', methods=["GET"])
+@login_required
+def get_confirmation(eq_id, form_type, collection_id):
+    navigator = get_questionnaire_manager(g.schema, g.schema_json).navigator
+    latest_location = navigator.get_latest_location(get_answers(current_user), get_completed_blocks(current_user))
+    if latest_location == 'confirmation':
+        metadata = get_metadata(current_user)
+        answers = get_answers(current_user)
+        schema_json = _render_schema(g.schema_json, answers, metadata)
+        return _render_template('confirmation', get_introduction_context(schema_json), rendered_schema_json=schema_json)
+    return redirect_to_questionnaire_page(eq_id, form_type, collection_id, latest_location)
```

Comment:
Should we refactor the 'check if the user is actually on summary/thank you/confirmation page or redirect' logic as we do it in a few places now